cli_builder/__init__.py

```diff
@@ -25,6 +25,18 @@ def my_cool_command_handler(args):
 import traceback
 
 
+class RawDefaultHelpFormatter(argparse.RawDescriptionHelpFormatter):
+    """A chimera of RawDescriptionHelpFormatter and ArgumentDefaultsHelpFormatter."""
+    def _get_help_string(self, action):
```

Comment:
Monkey patching private methods is not the route to nirvana.

Reply:
I think that:

```
class RawDefaultHelpFormatter(argparse.RawDescriptionHelpFormatter, argparse.ArgumentDefaultsHelpFormatter):
    pass
```

Is probably equivalent.

I think that making this user configurable makes sense though.  Will close and reopen on xsamtools.